presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -265,6 +265,28 @@ public void testDate()
         }
     }
 
+    @Test
+    public void testEnum()
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(postgreSqlServer.getJdbcUrl());
```

Comment:
Use `DataTypeTest` for this test. I convert JSONB tests to this form in https://github.com/prestosql/presto/pull/447

Reply:
Nvrm, this can't be tested with `DataTypeTest`, as you need to execute `CREATE TYPE`